### PR TITLE
Improve comments for RPC funcs

### PR DESCRIPTION
**Pull request by hsorellana**
**What this PR does / Why we need it**:
This PR adds some modifications to the comments of the Backfill funcs inside the FrontendService since they weren't so easy to read and understand its usage.

### CODE REVIEW

File: api/frontend.proto [hunk 1]
```diff
@@ -111,8 +111,7 @@ message GetBackfillRequest {
 // BETA FEATURE WARNING: This Request message is not finalized and still subject
 // to possible change or removal.
 message UpdateBackfillRequest {
-  // A Backfill object with SearchFields defined.
-  Backfill backfill_ticket = 1;
+  Backfill backfill = 1;
```
Comment (aLekSer) [inline api/frontend.proto:114]: I like this more as it is consistent with `CreateBackfillRequest`. 👍

File: api/frontend.swagger.json [hunk 2]
```diff
@@ -141,9 +125,9 @@
         ]
       },
       "delete": {
-        "summary": "Side effects: Any tickets waiting for this backfill will be returned to the active pool, no longer pending.",
+        "summary": "DeleteBackfill receives a backfill ID and deletes its resource.\nAny tickets waiting for this backfill will be returned to the active pool, no longer pending.",
         "description": "BETA FEATURE WARNING:  This call and the associated Request and Response\nmessages are not finalized and still subject to possible change or removal.",
-        "operationId": "FrontendService_DeleteBackfill",
```
Comment (aLekSer) [inline api/frontend.swagger.json:146]: Not sure, there are so many changes to `swagger.json` files. Do you use a different version of it?

**Review by aLekSer — COMMENTED** [review]
Changes in `proto` file is good. A bit worried of why `swagger.json` so significantly changed. Can you test this with `kind` and `make proxy-ui` to see if it looks good on `http://localhost:51500/`:
```
make create-kind-cluster
make get-kind-kubeconfig
make push-images
make install-chart
```

**Comment by hsorellana** [comment]
i used an online swagger editor to verify how the output json file looks like. And it looks like 
Screen Shot 2020-11-17 at 18 55 37

**Comment by aLekSer** [comment]
Tried to generate `swagger.json` files, they are the same as in this PR.

File: api/frontend.proto [hunk 3]
```diff
@@ -186,15 +186,19 @@ service FrontendService {
...
   // messages are not finalized and still subject to possible change or removal.
   rpc GetBackfill(GetBackfillRequest) returns (Backfill) {
     option (google.api.http) = {
       get: "/v1/frontendservice/backfills/{backfill_id}"
     };
   }
-
-  // Side effects: Any tickets waiting for this backfill will be returned to the active pool, no longer pending.
+  
+  // UpdateBackfill updates a Backfill object with an ID set and SearchFields defined.
```
Comment (Laremere) [inline api/frontend.proto:199]: extensions will be updated to.  Perhaps something like "UpdateBackfill updates search_fields and extensions for the backfill with the provided id"?

Comment (hsorellana) [reply api/frontend.proto]: Applied!

File: pkg/pb/evaluator.pb.go [hunk 4]
```diff
@@ -1,288 +1,160 @@
...
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
 // Code generated by protoc-gen-go. DO NOT EDIT.
-// versions:
-// 	protoc-gen-go v1.25.0
-// 	protoc        v3.10.1
```
Comment (hsorellana) [inline pkg/pb/evaluator.pb.go:18]: Look here @Laremere. When i run `make all-protos` to generate the changes from proto files, it deletes the licensing here. How should we proceed with this?? Do i leave it like it is or what?? Little help, please

Comment (Laremere) [reply pkg/pb/evaluator.pb.go:18]: If that's what the commands do - let's leave it like that.

**Review by Laremere — APPROVED** [review]

File: pkg/pb/evaluator.pb.go (hunk 4 above)
Comment (hsorellana) [reply pkg/pb/evaluator.pb.go:18]: ok, great!

**Review by aLekSer — APPROVED** [review]
Looks nice.

**Pull request merged**